src/allmydata/test/test_storage_http.py

```diff
@@ -442,6 +442,9 @@ def test_client_side_schema_validation(self):
             result_of(client.get_version())
 
     @given(length=st.integers(min_value=1, max_value=1_000_000))
+    # On Python 3.12 we're getting weird deadline issues in CI, so disabling
+    # for now.
```

Comment:
hmm, doesn't the "ci" Hypothesis profile already turn off the deadline? (or is "HealthCheck.too_slow" different from this deadline?)

Reply:
I don't know :(